Expensify/App
# Reduce the JS bundle size

~Holding on https://github.com/Expensify/react-native-onyx/pull/47~

### Details
This cuts the size of the initial JS the browser must download and parse by a lot

**Before:**
![Screen Shot 2021-02-12 at 2 30 16 PM](https://user-images.githubusercontent.com/32969087/107836644-ed82b480-6d41-11eb-90b5-d311c3138c2a.png)

**After:**
![Screen Shot 2021-02-12 at 3 45 00 PM](https://user-images.githubusercontent.com/32969087/107838365-a7315380-6d49-11eb-9b91-3d5e1e590435.png)

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/153796

### Tests
1. Make sure app builds on all platforms and otherwise functions normally. 
1. Make sure that pasting a chat from a google doc feature still works correctly.

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
